mlir/utils/jenkins/Jenkinsfile

```diff
@@ -198,7 +198,7 @@ String getLabelFromChip(String chip) {
         case "gfx906":
             return getLabelFromCodepath("vanilla")
         case "gfx908":
-            return "mlir && gfx908"
+            return "mlir && gfx908 && multi_gpu"
```

Comment:
Tuna's chief benefit is parallelism.  We have four gfx908s in the rocmhead pool, with 6, 4, 4, and 1 GPU respectively;  don't use the 1-GPU one.